lib/ui/geometry.dart

```diff
@@ -653,6 +653,17 @@ class Rect {
     );
   }
 
+  /// Returns a new rectangle which is the bounding box containing this
+  /// rectangle and the given rectangle.
+  Rect expand(Rect other) {
```

Comment:
Can you elaborate on the way this would be used? I'm trying to figure out if `expand` is the intuitive term in real uses.

Reply:
`expandToInclude`?

Reply:
Another option would be a constructor that takes a list of Rects.

`new Rect.boundingBox(<Rect>[...])`

Reply:
if it's to be a method, it could be `expandFor`